### DISCUSSION TITLE
Welcome! (Questions, Comments, Feedback)

### DISCUSSION THREAD

**Discussion by Everduin94** [Everduin94/better-commits#53] [Announcements]
## 👋 Hey there!

Every now and again better-commits sees a surge in new users/stars. I would love to hear your feedback!
- How did you find out about better-commits?
- What feature / improvement would you like to see in better-commits?

Thanks for taking the time to try out the tool and any feedback you may have :)

**Comment by nicolasmesser** [comment]
better-commits showed up in my "home feed" ... don't remember because someone I follow mentioned it or as "recommendation". Haven't used it yet but looking at implementing it with my next small project ... sounds like a great tool to have good commit history. Thank you for your work and for providing this tool!

**Comment by grandeljay** [comment]
It was a while ago but I think I saw a post on reddit or lemmy and thought it would help me learn and use conventional commits (it did!). Thank you for it!
